gateway-service/src/test/java/org/zowe/apiml/gateway/security/service/schema/ZoweJwtSchemeTest.java

```diff
@@ -238,34 +228,12 @@ void whenValidX509AuthSource_thenCommandIsNotExpired() {
         class NoCertificateInRequest {
             @Test
             void givenNoClientCertificate_thenCommandDoNotExpire() {
-                String errorHeaderValue = "ZWEAG160E No authentication provided in the request";
-                doReturn(errorHeaderValue).when(requestContext).get(AUTH_FAIL_HEADER);
-
-                ZoweJwtScheme.ZoweJwtAuthCommand command = (ZoweJwtScheme.ZoweJwtAuthCommand) scheme.createCommand(null, null);
-
-                assertNotNull(command);
-
-                Long expiration = (Long) ReflectionTestUtils.getField(command, "expireAt");
-                assertNotNull(expiration);
-                assertTrue(expiration <= System.currentTimeMillis());
+                assertThrows(AuthSchemeException.class, () -> scheme.createCommand(null, null));
```

Comment:
this looks like a wrong name for this test

Reply:
Renamed